src/components/duration/examples/duration-error/index.njk

```diff
@@ -0,0 +1,29 @@
+{% from "components/question/_macro.njk" import onsQuestion %}
+{% from "components/duration/_macro.njk" import onsDuration %}
+
+{% call onsQuestion({
+    "title": "How long did it take you complete this survey?",
```

Comment:
Perhaps update to a more realistic survey question, e.g.
```suggestion
    "title": "Approximately how long do you spend per day commuting to work?",
```
@LauraC119 can you advise on a decent question wording for an hours/mins question?

Reply:
@jrbarnes9 
How long do you spend travelling to and from work each day?